src/test_suite.js

```diff
@@ -12,6 +12,61 @@ TestSuite.prototype.name = function (name) {
   return this;
 };
 
+TestSuite.prototype.timestamp = function (timestamp) {
+  if(_.isDate(timestamp)) timestamp = timestamp.toISOString();
```

Comment:
[The specifications](https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd#L15) seem to support this, but without the milliseconds and the Z at the end. Is that an issue?